src/renderers/shared/utils/CallbackQueue.js

```diff
@@ -101,6 +102,4 @@ Object.assign(CallbackQueue.prototype, {
 
 });
 
-PooledClass.addPoolingTo(CallbackQueue);
-
-module.exports = CallbackQueue;
+module.exports = PooledClass.addPoolingTo(CallbackQueue);
```

Comment:
I'm more than happy to change it to something else, I'm just not sure what this something else is.